pkg/rke/k8s_service_options.go

```diff
@@ -526,12 +534,33 @@ func getKubeControllerOptions() map[string]string {
 	}
 }
 
+func getKubeControllerOptionsFor127() map[string]string {
```

Comment:
Comparing this map with the one returned by the existing `getKubeControllerOptions` method, it drops the pair `"pod-eviction-timeout": "5m0s",`. Why not reuse the existing method and handle the dropping in `getKubeControllerOptions127`?

Reply:
I am looking into it.